test/lib/godmin/resources/resource_service/ordering_test.rb

```diff
@@ -32,6 +27,12 @@ def test_apply_order_without_order
         @article_service.apply_order("", resources)
         assert_equal nil, resources.order_param
       end
+
+      def test_apply_order_with_custom_ordering_method
+        @article_service.apply_order("foobar_desc", @resources)
+        expected = { by_foobar: [@resources, "desc"] }
+        assert_equal expected, @article_service.called_methods[:ordering]
```

Comment:
The other service tests seem to be written like this: 
```ruby
assert_equal [@resources, "desc"], @article_service.called_methods[:ordering][:by_foobar]
```

Kinda the same thing, almost.. Should we write them all in the same way?